code/modules/projectiles/guns/ballistic.dm

```diff
@@ -188,6 +188,15 @@
 	if(handle_casings != HOLD_CASINGS)
 		chambered = null
 
+/obj/item/weapon/gun/projectile/proc/load_magazine(obj/item/A, mob/user)
```

Comment:
what have you done with this proc?

Reply:
this made the previous version of the shotgun work.